src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -36,6 +36,10 @@ internal sealed class SqlTableChangeMonitor<T> : IDisposable
         public const int LeaseIntervalInSeconds = 60;
         public const int LeaseRenewalIntervalInSeconds = 15;
 
+        private const string ChangeVersionColumnName = SqlTriggerConstants.WorkerTableChangeVersionColumnName;
```

Comment:
Why are you making new constants here? You can just use the ones from SqlTriggerConstants directly below.

Reply:
This is done to keep the SQL queries look concise and legible.

Reply:
It's fine to be a little verbose IMO - the difference of a dozen letters doesn't really matter.